src/entities/MeasurementType.ts

```diff
@@ -30,4 +31,10 @@ export class MeasurementType extends BaseEntity {
 		fridgeItemInfo => fridgeItemInfo.measurementTypeId
 	)
 	fridgeItemInfo: FridgeItemInfo;
+
+	@OneToMany(
```

Comment:
we'll probably merge this, so don't forget to change this too

Reply:
Fix my life